index.html

```diff
@@ -123,15 +123,9 @@ <h2 id="en-savoir-plus" class="text-primary pt-5 mb-5 fst-normal">
                 </h2>
 
                 <h2 class="h5">Qui peut se faire vacciner contre la Covid-19&nbsp;?*</h2>
-                <p>Toutes les personnes de plus de 18 ans résidant en France sont éligibles à la vaccination contre la Covid-19.</p>
-                <p>Par ailleurs, certaines personnes des moins de 18 ans peuvent d'ores et déjà se faire vacciner&nbsp;:</p>
-                <ul>
-                    <li>Les personnes de 16 à 18 ans inclus souffrant d’une pathologie à très haut risque de forme grave de Covid-19 (cancers et maladies hématologiques malignes en cours de traitement par chimiothérapie, maladies rénales chroniques sévères, transplantées d’organes solides, transplantées par allogreffe de cellules souches hématopoïétiques, poly-pathologies chroniques et présentant au moins deux insuffisances d’organes, <a href="https://solidarites-sante.gouv.fr/IMG/pdf/liste_maladies_rares_cosv_fmr-2.pdf" target="_blank" rel="noreferrer">certaines maladies rares et particulièrement à risque en cas d’infection</a>, trisomie 21, mucovisidose)&nbsp;;</li>
-                    <li>Les femmes enceintes à partir du 2e trimestre de grossesse&nbsp;;</li>
-                    <li>Les personnes de plus de 16 ans vivant dans le même foyer qu’une personne sévèrement immunodéprimée, enfant ou adulte (transplantés d’organes solides, transplantés récents de moelle osseuse récents, patients dialysés, patients atteints de maladies auto-immunes sous traitement immunosuppresseur fort de type anti-CD20 ou anti-métabolites)&nbsp;;</li>
-                 </ul>
-                <p class="fw-light fst-italic">Mis à jour le 31/05/2021</p>
-                <p class="text-danger fw-light fst-italic">* Ces informations sont susceptibles d'évoluer en fonction de la situation sanitaire. Tous les détails sont disponibles sur le site <a class="link-primary text-decoration-underline" href="https://solidarites-sante.gouv.fr/grands-dossiers/vaccin-covid-19/" target="_blank" rel="noreferrer">https://solidarite-sante.gouv.fr</a>.</p>
+                <p>Toutes les personnes de plus de 12 ans résidant en France sont éligibles à la vaccination contre la Covid-19.</p>
```

Comment:
```suggestion
                <p>Tous les adultes de plus de 18 ans résidant en France sont éligibles à la vaccination contre la Covid-19.</p>
                <p>Les adolescents de plus de 12 ans résidant en France sont également éligibles à la vaccination contre la Covid 19 avec l'accord de leurs parents ou responsables légaux (<a class="link-primary text-decoration-underline" href="https://eduscol.education.fr/2792/vaccination-des-jeunes-de-12-18-ans" target="_blank" rel="noreferrer">outils pour sensibiliser ce public à la vaccination</a>.</p>
```

Je pense qu'il faut ajouter la précision "accord avec les parents" pour les jeunes puisque c'est précisé sur le site du gouvernement.